lib/dogapi/v1/synthetic.rb

```diff
@@ -0,0 +1,34 @@
+require 'dogapi'
+
+module Dogapi
+  class V1 # for namespacing
+    # SyntheticService is the class responsible for dealing with the synthetics
+    class SyntheticService < Dogapi::APIService
+
+      API_VERSION = 'v1'
+
+      def synthetic(type, config, options = {})
```

Comment:
small nit, prefer to name this `create_synthetic` to keep in line with current library naming convention. You can refer to `dashboard` for structure, here: `https://github.com/DataDog/dogapi-rb/blob/0032f0b63ce8f356e5e0cea6a0d33fafc1c63bcf/lib/dogapi/v1/dashboard.rb#L31` .  

Additionally generally we lay out the required and optional args here in code comments above, but this endpoint's args are pretty large and verbose, so perhaps maybe just a comment pointing to the doc links for required/optional args. `https://docs.datadoghq.com/api/?lang=bash#create-a-test`

 thoughts? deferring to @dabcoder here

Reply:
We called it `create_test` in `datadogpy`: https://github.com/DataDog/datadogpy/blob/c880facaf8c79927ab2933b713d7dfa64d8ac86b/datadog/api/synthetics.py#L135 to be more specific about what the method actually does.